app/models/mapping.rb

```diff
@@ -231,7 +256,7 @@ def update_selected_terms(ids)
     self.selected_term_ids = ids
     return if spine.terms.any?
```

Comment:
Looks like at the edge case when it's freshly created spine and we're saving terms for the 2nd or more time spine terms and alignments will not be created. Not sure if it's critical and need to be fixed.

Reply:
It's been implemented this way for along time. Since nobody has complained, I think it's safe to leave it like this.